### ISSUE TITLE
A number of flying npc's do not have flight animation or paths

### ISSUE THREAD

**Issue by jerbookins** [TrinityCore/TrinityCore#19495]
[//]: # (**********************************)
[//]: # (** Fill in the following fields **)
[//]: # (**********************************)

**Description:**
A number of flying npc's do not have flight animation or paths

**Current behaviour:**
Quite a few NPCs, including this Nexus Guardian (entry 24276), seem to stand in mid-air instead of flying around or near their location.

**Expected behaviour:**
These in-flight NPCs should have some kind of flight path around their location and not stand still in mid air. Nexus Guardian is supposed to be in a pretty wide path going as far north east as Agmar's Hammer.

**Links**
- http://www.wowhead.com/npc=26276/nexus-guardian#comments

**Video**
- https://gfycat.com/GrossEveryAmericanriverotter

**Branch(es):**  3.3.5

**TC rev. hash/commit:**  TrinityCore rev. 3028be49296c 2017-04-20 15:44:21 +0200 (3.3.5 branch) (Win64, Release, Static)

**TDB version:** 335.63

**Operating system:** Windows 10 Server & Client

[//]: # (This template is for problem reports. For other types of report, edit it accordingly.)
[//]: # (For fixes containing C++, create a Pull Request.)

**Comment by jerbookins** [comment]
I will update with other NPCs that seem to have this same behavior.

**Comment by Aokromes** [comment]
this bug is still valid?

**Comment by jerbookins** [comment]
@Aokromes yes it's still valid. There have been a lot of improvements to flying creatures, particularly around Coldarra and Wyrmrest Temple but the specific instances are still behaving the same.

The Nexus Guardian in the video is located at Moonrest Gardens in Dragonblight and the Anub'ar Blightbeasts shown are standing in mid-air above Azjol Nerub in Dragonblight.

I don't have any other recorded instances of this behavior but I can search for more if necessary.

**Comment by jerbookins** [comment]
![wowscrnshot_090918_212749](https://user-images.githubusercontent.com/16745918/45274983-55e55180-b477-11e8-8468-a77cf4fd89ca.jpg)

Scorchscale Drake (42 elite), near Onyxia's cave in Dustwallow Marsh

**Comment by Killyana** [comment]
This is caused by your client changes.

**Comment by shamage** [comment]
Is not it enough to set WP only on each NPC alone?

**Comment by jerbookins** [comment]
@Killyana in that particular instance, you're correct. I should have tested on the normal client.

But it's understandable why I might assume it isn't a problem caused by custom models.

https://streamable.com/edit/dlx80

**Comment by Faq** [comment]
```
-- Anubar Blightbeast wont go under texture
UPDATE `creature_template` SET `flags_extra` = `flags_extra`| 0x200 WHERE `entry` IN (26607);
```

And creature id 26276 with guid 111305, looks like missing flypath as other 2 have it.

**Comment by offl** [comment]
Creature 26276 requires paths or maybe only for guid 111305
Positions & paths for other 2 creatures looks weird since they spawns in trees
